packages/components/pages/f-contact-preferences/src/components/_tests/ContactPreferences.test.js

```diff
@@ -53,11 +57,9 @@ const mountContactPreferences = ({
     mocks = sutMocks,
     propsData = sutProps,
     data = dataDefaults,
-    storeOverride = null,
     initialiseOverride = null
 } = {}) => {
-    const store = storeOverride || createStore({ state, actions });
-    store.registerModule = registerStoreModuleSpy;
+    const store = createStore({ state, actions });
 
     if (initialiseOverride) {
         ContactPreferences.methods.initialise = initialiseOverride;
```

Comment:
Just spotted this in `f-account-info` once `ContactPreferences.methods.initialise` has been overridden it is never reset to the original method.